### ISSUE TITLE
[Feature] ZIP Gallery Metadata File

### ISSUE THREAD

**Issue by alucicrazy** [stashapp/stash#4735]
**Is your feature request related to a problem? Please describe.**
After filling  the Gallery data, it is saved only to the stash database

**Describe the solution you'd like**
Something like `ComicInfo.xml` but for all types of galleries

**Additional context**
[ComicInfo.xml Community run schema](https://github.com/anansi-project/comicinfo)

**Comment by DogmaDragon** [comment]
You can export existing Stash metadata into JSON. 
https://docs.stashapp.cc/in-app-manual/tasks/jsonspec/

**Comment by alucicrazy** [comment]
Yeah, but what I am talking about is about having the metadata info
embedded in the zip file, being editable and using it when Stash is
scanning zip files to fill the gallery information

**Comment by DogmaDragon** [comment]
Embedding data would change the .zip checksum, which is not ideal.
